torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -141,6 +141,67 @@ bool isSupported(Node* node) {
   return false;
 }
 
+// Code to detect specializations for testing CustomPasses.
+static std::unique_ptr<RegisterPass> tTSpecializationDetectionPass = nullptr;
```

Comment:
Code for testing shouldn't leak into our actual implementation files, especially when we have an open registration mechanism.

Reply:
Which file do you think the code should go into instead then?

Reply:
register a custom pass in a test file